api/turing/imagebuilder/ensembler.go

```diff
@@ -0,0 +1,42 @@
+package imagebuilder
```

Comment:
I feel like the `imagebuilder` package could be like the `cluster` package, that simply defines the generic Kaniko related functionality. This file can go into some ensembler related package. wdyt ?